### PR TITLE
chore(other): added focus related theme fallbacks

### PR DESCRIPTION
**Pull request by daragh-king-genesys**
added focus related theme fallbacks

✅ Closes: COMUI-3119

### CODE REVIEW

File: packages/genesys-spark-components/src/components/beta/gux-calendar-beta/components/single/gux-calendar.scss [hunk 1]
```diff
@@ -132,8 +132,15 @@
 
       &:focus-visible {
         border-radius: var(--gse-ui-calendarMenu-month-focusBorderRadius);
-        outline: var(--gse-semantic-focusRing-default-border-width) solid
-          var(--gse-semantic-focusRing-default-border-color);
+        outline: var(
+            --gse-semantic-focusRing-default-border-width,
+            --gse-semantic-focusOutline-md-borderWidth
```
Comment (321gillian) [inline packages/genesys-spark-components/src/components/beta/gux-calendar-beta/components/single/gux-calendar.scss:137]: the preferred token is 3px and the fallback is 2px. that correct?

File: packages/genesys-spark-components/src/components/beta/gux-month-picker/gux-month-calendar/gux-month-calendar.scss [hunk 2]
```diff
@@ -50,14 +50,24 @@
...
+              --gse-semantic-focusRing-offset,
+              --gse-semantic-focusOutline-offset
             )
         );
-        outline: var(--gse-semantic-focusRing-width)
-          var(--gse-semantic-focusRing-default-border-style)
+        outline: var(
+            --gse-semantic-focusRing-width,
+            --gse-semantic-focusOutline-md-borderWidth
+          )
```
Comment (321gillian) [inline packages/genesys-spark-components/src/components/beta/gux-month-picker/gux-month-calendar/gux-month-calendar.scss:64]: same thing here, 3px and 2px

File: packages/genesys-spark-components/src/components/beta/gux-calendar-beta/components/single/gux-calendar.scss (hunk 1 above)
Comment (daragh-king-genesys) [reply packages/genesys-spark-components/src/components/beta/gux-calendar-beta/components/single/gux-calendar.scss:137]: @VirtualClover is this change corect?

**Review by 321gillian — APPROVED** [review]

File: packages/genesys-spark-components/src/components/stable/gux-tag/gux-tag.scss [hunk 3]
```diff
@@ -16,7 +16,7 @@
   padding: var(--gse-ui-tag-padding);
   font-family: var(--gse-ui-tag-text-fontFamily);
   font-size: var(--gse-ui-tag-text-fontSize);
-  font-weight: var(--gse-ui-tag-text-fontWeight, 600);
+  font-weight: var(--gse-ui-tag-text-fontWeight, var(600));
```
Comment (321gillian) [inline packages/genesys-spark-components/src/components/stable/gux-tag/gux-tag.scss:19]:
```suggestion
  font-weight: var(--gse-ui-tag-text-fontWeight, 600);
```

File: packages/genesys-spark-components/src/components/stable/gux-rating/gux-rating.scss [hunk 4]
```diff
@@ -9,8 +9,12 @@
...
   @include focus.gux-focus-ring-small;
 
-  outline-offset: var(--gse-semantic-focusRing-offset);
-  box-shadow: 0 0 0 1px var(--gse-semantic-focusRing-small-border-style);
+  outline-offset: var(
+    --gse-semantic-focusRing-offset,
+    var(--gse-semantic-focusOutline-offset)
+  );
+  box-shadow: 0 0 0 1px
+    var(--gse-semantic-focusRing-small-border-style, var(solid));
```
Comment (321gillian) [inline packages/genesys-spark-components/src/components/stable/gux-rating/gux-rating.scss:17]:
```suggestion
    var(--gse-semantic-focusRing-small-border-style, solid);
```

File: packages/genesys-spark-components/src/components/stable/gux-tag/gux-tag.scss (hunk 3 above)
Comment (daragh-king-genesys) [reply packages/genesys-spark-components/src/components/stable/gux-tag/gux-tag.scss]: fixed

File: packages/genesys-spark-components/src/components/stable/gux-rating/gux-rating.scss (hunk 4 above)
Comment (daragh-king-genesys) [reply packages/genesys-spark-components/src/components/stable/gux-rating/gux-rating.scss]: fixed

File: packages/genesys-spark-components/src/components/stable/gux-tag/gux-tag.scss [hunk 5]
```diff
@@ -74,9 +74,15 @@
...
-        border-radius: var(--gse-semantic-focusRing-cornerRadius);
-        outline: var(--gse-semantic-focusRing-small-border-width) solid
-          var(--gse-semantic-color-focus, --gse-ui-color-focus);
+        border-radius: var(
+          --gse-semantic-focusRing-cornerRadius,
+          var(--gse-semantic-focusOutline-sm-borderRadius)
+        );
+        outline: var(
+            --gse-semantic-focusRing-small-border-width,
+            var(--gse-semantic-border-focus)
```
Comment (321gillian) [inline packages/genesys-spark-components/src/components/stable/gux-tag/gux-tag.scss:83]: This is a colour fallback for a width token.

Comment (daragh-king-genesys) [reply packages/genesys-spark-components/src/components/stable/gux-tag/gux-tag.scss]: fixed

**Review by gavin-everett-genesys — APPROVED** [review]

**Review by 321gillian — APPROVED** [review]

**Pull request merged**